spec/jobs/alert_mailer_job_spec.rb

```diff
@@ -39,7 +39,7 @@
     allow_any_instance_of(AlertMailerJob).to receive(:provider_job_id) { job_id }
 
     allow_any_instance_of(AlertMailerJob).to receive(:subscription) { subscription }
-    expect(AlertMailerJob.queue_adapter).to receive(:enqueue).twice.ordered
+    expect(AlertMailerJob.queue_adapter).to receive(:enqueue).exactly(3).times.ordered
```

Comment:
Ok, how the changes here changes this test?

Reply:
[BroiSatse](https://github.com/BroiSatse) have a look at the commit message for this line of code 75e22db667f8c5ed5aae14e35f580c8c50363ff9 👀